File: Python/card.py
from payment import Payment
class Card(Payment):
    """
    Card class
    """
    card_number  = int
    card_date    = str
    card_cvc     = int 

    def __init__(self,id,amount,card_number,card_date,card_cvc):
        super().__init__(id,amount)
        self.card_number = card_number
        self.card_date   = card_date
        self.card_cvc    = card_cvc

File: Java/Card.java
package Java;

/**
 * Card
 */
public class Card extends Payment{

    Integer cardNumber;
    String  cardDate;
    Integer cardCvv;

    public Card(Integer id,Float amount,Integer cardNumber,String cardDate, Integer cardCvv){
        super(id, amount);
        this.cardNumber = cardNumber;
        this.cardDate = cardDate;
        this.cardCvv = cardCvv;
    }
    
}